geps/gep-3155/index.md

```diff
@@ -63,19 +63,34 @@ type GatewayTLSConfig struct {
     ...
 }
 type GatewayBackendTLS struct {
-  // ClientCertificateRef is a reference to an object that contains a Client
-  // Certificate and the associated private key.
+  // ClientCertificateRef references an object that contains a client certificate 
+  // and its associated private key. It can reference standard Kubernetes resources,
+  // i.e., Secret, or implementation-specific custom resources.
   //
-  // References to a resource in different namespace are invalid UNLESS there
-  // is a ReferenceGrant in the target namespace that allows the certificate
-  // to be attached. If a ReferenceGrant does not allow this reference, the
-  // "ResolvedRefs" condition MUST be set to False for this listener with the
-  // "RefNotPermitted" reason.
+  // This setting can be overridden on the service level by use of BackendTLSPolicy.
```

Comment:
This statement is no longer true. gateway client certificate is part of gateway identity and should be the same for all connections from gateway to backend, this setting cannot be overridden on service level

Reply:
I was thinking the same, but I didn’t want to remove it without discussing it first. If everyone agrees, I’ll go ahead and remove it.